### ISSUE TITLE
Non-damus apps quoting a damus user does not generate notifications in Damus due to lack of p-tag

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#54]
## client initiating action
primal

## client receiving or observing action (or lack thereof)
damus

## what happens 
Damus user does not receive notification of a quote post created by other app, due to not using p tag in quotes.

## suggestion
Use p tag in quotes

**Comment by alltheseas** [comment]
Does primal add p tags in quotes @mbrat1 ?
Does nostur add p tags in quotes @fabianfabian ?

This would be a straightforward test case for nostrCI to auto verify all nostr apps use p tags in quotes, if this is considered best practice https://github.com/nostrCI/nostrCI/issues/6

**Comment by fabianfabian** [comment]
> Does nostur add p tags in quotes @fabianfabian ?

yes but only when tapping the quote button on a post. When manually pasting nostr:note1 or nostr:nevent1 it does not, I should probably add that.

**Comment by alexgleason** [comment]
Clients don't have to notify about quotes. You p tag if you want the recipient to be notified, and you don't if you don't.

**Comment by fiatjaf** [comment]
Whatever @alexgleason said, it's the choice of whoever is sending the thing (by which I mean they chose a client that does what they want).

**Comment by jb55** [comment]
yeah I can see why some clients might not want to notify of quote reposts, you can still look them up via the q tag if you want to list them.

**Comment by alltheseas** [comment]
Choice is great for the customer.

@fabianfabian and devs: do yall think the notification choices as outlined by Fabian in nostur should be consistent in one app? 

Reading the original report, it seems customers are expecting a certain behavior (in this case, I believe it stems from twitter). 

From here flows a design consideration and open question: how does a customer learn about the idiosyncrasies of nostr apps X, Y, and Z? 

If it all, how might the dev instruct the customer about the design patterns a particular app might take? 

For instance during onboarding, or before insrall describe:
Attention grabbing App X: customer is informed of twitter constant algo feed, always notifications
Sleepier App Y: notifications off by default, no algo

@robagreda

@elidyweaver 

@karnagebitcoin

**Comment by alexgleason** [comment]
I think if devs don't have an opinion, they should add the "p" tag because it's what most users expect. However I've encountered some Japanese devs on the Fediverse that specifically don't want to notify with quote posts.

Clients could give users the option when creating the event. But generally they should just add the "p" tag, and if they don't, users will just be confused by it forever when it happens rarely, and that's okay.

**Comment by jb55** [comment]
On Thu, Aug 15, 2024 at 12:39:17PM GMT, Alex Gleason wrote:
>I think if devs don't have an opinion, they should add the "p" tag because it's what most users expect. However I've encountered some Japanese devs on the Fediverse that specifically don't want to notify with quote posts.
>
>Clients could give users the option when creating the event. But generally they should just add the "p" tag, and if they don't, users will just be confused by it forever when it happens rarely, and that's okay.

should probably just have an option to enable/disable like we do with
reply p tags.
